README.md

```diff
@@ -101,6 +101,13 @@ Other options for `install`:
 * `--k3s-extra-args` - Optional extra arguments to pass to k3s installer, wrapped in quotes, i.e. `--k3s-extra-args '--no-deploy traefik'` or `--k3s-extra-args '--docker'`. For multiple args combine then within single quotes `--k3s-extra-args '--no-deploy traefik --docker'`.
 * `--k3s-version` - set the specific version of k3s, i.e. `v0.9.1`
 - `--ipsec` - Enforces the optional extra argument for k3s: `--flannel-backend` option: `ipsec`
+* `-d=sql, --datastore=sql` - Use an external MySQL datastore for HA
+* `-d=postgresql, --datastore=postgresql` - Use an external PostgreSQL database for HA
+* `--db-hostname` - Hostname of external database server (Used with either `--ha-mysql` or `--ha-psql`)
```

Comment:
This comment is out of date now

Reply:
Yes I have tested it out with MySQL and PostgreSQL. I'm adding those and updating the testing screenshots in the description above.